components/expenses/PayExpenseModal.js

```diff
@@ -94,8 +95,12 @@ const getTotalPayoutAmount = (expense, { paymentProcessorFee, feesPayer }) => {
   }
 };
 
-const canCustomizeFeesPayer = payoutMethodType => {
-  return [PayoutMethodType.BANK_ACCOUNT].includes(payoutMethodType);
+const canCustomizeFeesPayer = (expense, collective, hasManualPayment, feeAmount) => {
+  return Boolean(
+    [PayoutMethodType.BANK_ACCOUNT].includes(expense.payoutMethod?.type) && // Only for transferwise at the moment
+      expense.amount === get(collective, 'stats.balanceWithBlockedFunds.valueInCents') && // Only when emptying the collective balance
```

Comment:
I didn't test. Is expense.amount an API v1 object?

Reply:
Unfortunately, `expense.amount` was initially implemented as an Integer, not an amount. That's something we need to refactor: https://github.com/opencollective/opencollective/issues/3231